### PR TITLE
Merge 3.3.x to main

### PR DESCRIPTION
**Pull request by nickva**
Bring in 3.3.x branch changes into main.

### CODE REVIEW

**Review by jaydoane — APPROVED** [review]

File: dev/run [hunk 1]
```diff
@@ -475,7 +475,6 @@ def boot_nouveau(ctx):
         "server",
         "nouveau.yaml",
     ]
-
```
Comment (pgj) [inline dev/run:478]: Was not this an unintended change?  If not, why?

File: src/docs/src/whatsnew/3.3.rst [hunk 2]
```diff
@@ -83,6 +83,7 @@ Features and Enhancements
 * :ghissue:`4347`: Remove failed ``couch_plugins`` experiment, fixes more
   ``undef`` errors.
 * :ghissue:`4348`: Fix ``undef`` error in ``weatherreport``.
+* :ghissue:`4353`: Allow starting of more than one replication job. (D’OH!)
```
Comment (pgj) [inline src/docs/src/whatsnew/3.3.rst:86]: "D'OH"?  Should not the release notes stick to a more formal technical style?  I am just curious.

File: dev/run (hunk 1 above)
Comment (nickva) [reply dev/run:478]: It just showed as a diff between the 3.3.x branch and I thought'd bring it over to minimize differences.

File: src/docs/src/whatsnew/3.3.rst (hunk 2 above)
Comment (nickva) [reply src/docs/src/whatsnew/3.3.rst:86]: Nah, we like to be a bit informal and whimsical with our release notes. They often reflect the personality of whoever writes them.  We have [Star Trek references](https://docs.couchdb.org/en/stable/whatsnew/3.3.html#highlights), [Rainbow Gnomes](https://docs.couchdb.org/en/stable/whatsnew/3.1.html#features-and-enhancements), [Party cats](https://docs.couchdb.org/en/stable/whatsnew/3.0.html#upgrade-notes) and other such things.

File: dev/run (hunk 1 above)
Comment (pgj) [reply dev/run:478]: Ah, now it makes sense.

File: src/docs/src/whatsnew/3.3.rst (hunk 2 above)
Comment (pgj) [reply src/docs/src/whatsnew/3.3.rst:86]: Thanks for the explanation :-)

**Pull request merged**